scripts/quickstart.sh

```diff
@@ -2,6 +2,9 @@
 
 set -e
 
+OLDIFS=$IFS
```

Comment:
Can you explain what's this?

Reply:
It reads the current IFS so it can be restored at the end of the script (so it doesn't other scripts a validator may be running)